@AH-64D Apache Longbow/Addons/fza_ah64_controls/config/cfgVehicles.hpp

```diff
@@ -242,6 +243,10 @@ class CfgVehicles
 		#include "cfgVehicles\ace.hpp"
 		#include "sensor_b2e.hpp"
 
+		//SFM Variables-------------/
+		flightModel			   = "SFMPlus";
```

Comment:
Please can you rename `flightModel` to `fza_flightModel`?

Reply:
Complete